examples/11-mailbox-agents/agent1.py

```diff
@@ -0,0 +1,38 @@
+from uagents import Agent, Context, Model
+from uagents.resolver import RulesBasedResolver
+
+
+class Message(Model):
+    message: str
+
+
+AGENT1_ADDRESS = "agent1qv2l7qzcd2g2rcv2p93tqflrcaq5dk7c2xc7fcnfq3s37zgkhxjmq5mfyvz"
+AGENT2_ADDRESS = "agent1qv73me5ql7kl30t0grehalj0aau0l4hpthp4m5q9v4qk2hz8h63vzpgyadp"
+RELAY_SERVER_URL = "http://127.0.0.1:8000"
+
+agent = Agent(
+    name="alice",
+    port=8010,
+    seed="agent1 secret phrase",
+    use_relay=True,
+    resolve=RulesBasedResolver(
+        {
+            AGENT1_ADDRESS: f"{RELAY_SERVER_URL}/v1/submit",
+            AGENT2_ADDRESS: f"{RELAY_SERVER_URL}/v1/submit",
+        }
+    ),
```

Comment:
My thought process for the configuration something like this:

```python
Agent(
    ...
    mailbox='<api key>',
)
```

or for the longer version:

```python
Agent(
    ...
    mailbox='<api key>@<mailbox base url>',
)
```

or the full version

```python
Agent(
    ...
    mailbox={
        'base_url': '<mail box base url>',
        'api_key': '<api key>',
    }
)
```

Reply:
OK, `api_key` is not implemeted yet, right? Or did I miss that?